src/Files.App/Views/Pages/Properties.xaml.cs

```diff
@@ -107,18 +112,14 @@ private void NavigationView_SizeChanged(object? sender, SizeChangedEventArgs? e)
 					}
 				}
 			}
+
+			// Sets properties window drag region.
 			appWindow.TitleBar.SetDragRectangles(new RectInt32[]
```

Comment:
Does this work on Windows 10? @gave92

Reply:
Nope, SetDragRectangles has currently no effect on win10. But I'm still able to drag the window from the area around the close button. Good enough till they fix it, using the same workaround we added in MainPage.cs would be overly complex I guess.

Reply:
Sounds good, we can leave as is